lib/abci/handlers/commitHandlerFactory.js

```diff
@@ -65,6 +104,65 @@ function commitHandlerFactory(
       blockExecutionContext.reset();
     }
 
+    rootTree.rebuild();
```

Comment:
Let's add a comment, like "rebuild root tree with committed data from the current block"